Automattic/Gravatar-SDK-iOS
# Adding inline docs for `ProfileViewConfiguration`

Closes #

### Description

This PR adds inline docs to the `ProfileViewConfiguration` structure.
I also took the chance to wrap the avatar related configurations in an `AvatarConfiguration` structure. This is common pattern among `UIContentConfiguration`.

### Testing Steps

- Check that the descriptions written are good and there are no typos.
- Build docs (product -> Build documentation)
  - Search for ProfileViewConfiguration.
  - Check that it looks all right.
